src/Me/Navigation/Navbar.tsx

```diff
@@ -20,24 +23,42 @@ const MenuItem = ({
 }: {
   icon: string;
   label: string;
-  to: string;
+  to?: string;
+  onClick?: () => void;
 }) => {
   const router = useRouter();
+  if (to) {
+    return (
+      <Link href={to}>
+        <NavItemDecoration
+          className={classNames({ active: router.pathname === to })}
+        >
+          <Icon />
+          <Label>{label}</Label>
+        </NavItemDecoration>
+      </Link>
+    );
+  }
   return (
-    <Link href={to}>
-      <NavItemDecoration
-        className={classNames({ active: router.pathname === to })}
-      >
+    <button onClick={toggleChat}>
```

Comment:
Since you pass `onClick`, did you mean 
```suggestion
    <button onClick={onClick}>
```
?

Reply:
ive done this